controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -69,6 +69,8 @@
                 context = context.withValue(AUTH_CONTEXT_PARAMS, paramMap);
                 context = context.withValue(INTERCEPTOR_OBJECT, this);
             }
+        } else {
+            call.close(Status.fromCode(Status.Code.UNAUTHENTICATED), headers);
         }
```

Comment:
Should we be returning `null` like in line 67?

Reply:
Fixed.